packages/gatsby-plugin-sitemap/src/internals.js

```diff
@@ -36,6 +36,7 @@ export const defaultOptions = {
   }`,
   output: `/sitemap.xml`,
   exclude: [`/dev-404-page`, `/404`, `/offline-plugin-app-shell-fallback`],
+  addLinkToHead: true,
```

Comment:
To keep things consistent with the rest of Gatsby, could you change this to `createLinkToHead` https://www.gatsbyjs.org/docs/api-specification/#operators

Reply:
@KyleAMathews Sure, maybe `createLinkInHead` would be better though?

Reply:
Yeah, even better :-)

Reply:
@KyleAMathews https://github.com/gatsbyjs/gatsby/pull/4641